src/components/Slide/player.css

```
@@ -396,17 +397,214 @@
   transition: opacity 160ms ease;
 }
 
+.slide-interaction-overlay--large {
+  --mdf-slide-font-size: 24px;
+  --mdf-slide-line-height: 36px;
+  --text-sm-font-size: 21px;
+  --text-sm-line-height: 30px;
+  --text-xs-font-size: 21px;
+  --text-xs-line-height: 28px;
+  display: flex;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)